File: backend/src/main/java/com/joying/chat/dto/FileUploadResponse.java
package com.joying.chat.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

/**
 * 파일 업로드 결과.
 */
@Getter
@Builder(toBuilder = true)
@NoArgsConstructor
@AllArgsConstructor
public class FileUploadResponse {

	private Long fileId;
	private String url;
	private String fileName;
	private long fileSize;
	private FileType fileType;
}

File: backend/src/main/kotlin/com/joying/chat/dto/FileUploadResponse.kt
package com.joying.chat.dto

/**
 * 파일 업로드 Response
 */
data class FileUploadResponse(
    val fileId: Long,
    val url: String,
    val fileName: String,
    val fileSize: Long,
    val fileType: FileType
)

enum class FileType {
    IMAGE,
    FILE
}